openapi-cli/src/test/resources/ballerina-to-openapi/expected_gen/snowpeak_request_body_ref.yaml

```diff
@@ -44,6 +44,38 @@ paths:
             text/plain:
               schema:
                 type: string
+  /reservations:
+    post:
+      summary: Represents Snowpeak reservations resource
+      operationId: postReservations
+      requestBody:
+        description: Array representation of Reservations
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/Reservation'
```

Comment:
was this `Reservation` component schema generated under the component schema?

Reply:
Yes, I have also verified the generated OAS with swagger editor